src/components/RadioConditionalInput.tsx

```diff
@@ -329,6 +334,7 @@ const renderConditionalRadioButton = (
             <div className="govuk-radios__item">
                 {radio.inputErrors.length > 0 ? radioInputWithError : baseRadioInput}
                 {radioLabel}
+                {radioButtonHint}
```

Comment:
Does this `radioButtonHint` want to be rendered conditionally?

Reply:
good call have made the change on this